core/src/data.ts

```diff
@@ -212,6 +215,7 @@ export class DrumRollConverter extends DrumsConverter {
         }
       }
     }
+    roll.dispose();
```

Comment:
Not that familiar with how this stuff works in Javascript but it seems very strange to dispose a function parameter inside the function.

Reply:
indeed! i'd already fixed this but nice catch!